<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>261</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>Videos publicitarios</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QListView" name="videoListView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>251</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bAgregarVideo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bEliminarVideo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>30</y>
     <width>671</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>Vista previa </string>
   </property>
   <widget class="QPushButton" name="bPlay">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>500</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bStop">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bPausa">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pausa</string>
    </property>
   </widget>
   <widget class="QFrame" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>661</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>951</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="bIniciarPrograma">
    <property name="geometry">
     <rect>
      <x>813</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar Programa</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
